templates/title.mustache

```diff
@@ -20,16 +20,16 @@
 				</a>
 			</li>
 			<li>
-			<a class="github-button nav-social" href="https://github.com/canjs/{{name}}"
-				data-count-href="/canjs/{{name}}/stargazers"
-				data-count-api="/repos/canjs/{{name}}#stargazers_count">Star</a>
-	  	</li>
+				<a class="github-button nav-social" href="https://github.com/canjs/{{repoName}}"
+				   data-count-href="/canjs/{{repoName}}/stargazers"
+				   data-count-api="/repos/canjs/{{repoName}}#stargazers_count">Star</a>
+			</li>
 		</ul>
 	{{/if}}
 	{{#with (getClosestWithPackage)}}
 		<ul class="title-links">
 			<!-- <li><a href="#">docco</a></li> -->
-			{{#with (sourceLink .)}}<li><a href="{{.}}">Edit on GitHub</a></li>{{/with}}
+			<li><a href="{{sourceLink (repoName)}}">Edit on GitHub</a></li>
```

Comment:
No, I don't think it would. However I haven't been able to reproduce a situation where `sourceLink` fails to get a link. 

That would only occur if `docMapInfo.getCurrent().src` is undefined and I haven't seen that. Do you know if that's a likely situation?